Library/Formula/gdub.rb

```diff
@@ -0,0 +1,12 @@
+class Gdub < Formula
+  desc "Gdub is a Gradle wrapper wrapper"
+  homepage "http://gdub.rocks"
+  url "https://github.com/dougborg/gdub/archive/v0.1.0.tar.gz"
+  sha256 "ddf2572cc67b8df3293b1707720c6ef09d6caf73227fa869a73b16239df959c3"
+
+  depends_on "gradle"
+
+  def install
+    bin.install "bin/gw"
+  end
```

Comment:
Yeah I tried that but with no success. The purpose of this shell script is to be able to run the gradlew script from any folder of the project as opposed to being able to do it only from the root folder. I don't see a way to write a test without having to setup a whole gradle project. I tried with a `gradle init` and then use that, but the testing environment did not seem to like that.